component/site/views/default/helpers/defaulteventmanagementdialog.php

```diff
@@ -50,11 +51,11 @@ function DefaultEventManagementDialog($view, $row, $mask, $bootstrap = false)
 		$editImg      = HTMLHelper::image('com_jevents/icons-32/edit.png', Text::_("EDIT_EVENT"), null, true);
 		$editLink     = $row->editLink();
 		//$editLink     = $popup ? "javascript:jevEditPopupNoHeader('" . $editLink . "');" : $editLink;
-		$editLink     = $popup ? "javascript:jevModalNoHeader('myEditModal','" . $editLink . "');" : $editLink;
+		$editLink     = $popup ? "javascript:jevModalNoHeader('myEditModal','" . $editLink . "');javascript:closeJevModalBySelector('#" . $actionId . "');" : $editLink;
```

Comment:
does this really work with javascript: repeated??
It should just be 
`jevModalNoHeader('myEditModal','" . $editLink . "');closeJevModalBySelector('#" . $actionId . "');`

repeated below